app/src/main/java/com/github/se/cyrcle/ui/gambling/GamblingScreen.kt

```diff
@@ -245,21 +249,52 @@ fun WheelView(
 }
 
 @Composable
-fun GamblingScreen() {
+fun GamblingScreen(navigationActions: NavigationActions, userViewModel: UserViewModel) {
+  val userState by userViewModel.currentUser.collectAsState()
+  val coins = userState?.details?.wallet?.getCoins()
+  val spinCost = 10 // Cost to spin the wheel
+
   Box(
       modifier = Modifier.fillMaxSize().testTag("gambling_screen"),
       contentAlignment = Alignment.Center) {
         var wheelSpinFunction by remember { mutableStateOf<(() -> Unit)?>(null) }
 
+        Text(
+            text = "Coins: ${coins ?: "BROKIE"}",
```

Comment:
I don't like saying this but this whole text should be in string.xml

Reply:
Even worse, I shouldnt have left this in here haha. Ill add a string.xml and make it appropriate.